.env

```diff
@@ -34,4 +34,5 @@ _APP_FUNCTIONS_CPUS=1
 _APP_FUNCTIONS_MEMORY=128
 _APP_FUNCTIONS_MEMORY_SWAP=128
 _APP_MAINTENANCE_INTERVAL=86400
-_APP_SYSTEM_RESPONSE_FORMAT=
\ No newline at end of file
+_APP_SYSTEM_RESPONSE_FORMAT=
+_APP_USAGE_STATS=
```

Comment:
Should probably be 'enabled' and not empty

Reply:
updated